pdr_backend/pdr_dashboard/util/db.py

```diff
@@ -422,25 +422,33 @@ def predictoors_metrics(self) -> dict[str, Union[int, float]]:
                     SUM(
                         CASE WHEN p.payout > p.stake
                         THEN p.payout - p.stake ELSE 0 END
-                    ) AS tot_gross_income
+                    ) AS tot_gross_income,
+                    SUM(CASE WHEN p.payout = 0 THEN p.stake ELSE 0 END) AS stake_loss,
                 FROM
                     {tbl_parquet_path(self.lake_dir, BronzePrediction)} p
             """
         if self.start_date_ms:
             query_predictoors_metrics += f" WHERE timestamp > {self.start_date_ms}"
-        predictoors, avg_accuracy, tot_stake, tot_gross_income = (
+        predictoors, avg_accuracy, tot_stake, tot_gross_income, stake_loss = (
             self.file_reader._query_db(
                 query_predictoors_metrics,
                 scalar=True,
                 cache_file_name="predictoor_metrics_predictoors",
             )
         )
 
+        profit = (
+            tot_gross_income
+            - (tot_stake * 2 * self.fee_cost if tot_stake else 0)
```

Comment:
the costs calculation looks wrong. It should be nr_of_predictions * 2 * fee_costs